### ISSUE TITLE
Mention a user in a comment

### ISSUE THREAD

**Issue by wittwerch** [directus/directus#1620]
## Version Info
  - Directus version and branch: 61e0e0268f646e5879c48296b80625cc7fe3a10b
  - PHP version: 7.1
  - MySQL version: 5.7
  - Web server: Apache
  - OS name and version: Docker 

## Expected Behavior
Typing the at-symbol to get a list of users like on Github..

## Actual Behavior
Start typing
![screen shot 2017-06-23 at 10 37 09](https://user-images.githubusercontent.com/1575445/27473583-f2069716-57ff-11e7-9c51-e4dedd72b238.png)

Select the profile picture and click on it, it turns to "undefined".
![screen shot 2017-06-23 at 10 36 55](https://user-images.githubusercontent.com/1575445/27473586-f5570f7c-57ff-11e7-93d8-9289b047983f.png)

Maybe the dropdown should be smaller and right below the at-symbol, like on Github
And it would be cool if a selection appears without actually typing only the at-symbol.

Basically copy the behaviour from Github :)

**Comment by benhaynes** [comment]
1. We need to clean up the dropdown of users as you type the @mention
2. We need to resolve the `undefined undefined` issue

**Comment by WellingGuzman** [comment]
2. Closed by https://github.com/directus/directus/commit/b49c34342f436112d895fe92a6f79df77938deec

**Comment by wittwerch** [comment]
The selection now works. However I agree with @benhaynes that the dropdown needs a little rework.

- As you type the at-symbol, a list should pop off even when you didn't enter the next char
- The list should be smaller
- You should be able to navigate with the cursor to select an entry
- Allow the enter button to be used to select a value :)  see #1621

**Comment by benhaynes** [comment]
Agree on all points. We will be focusing in other areas of the platform over the next 2 months – perhaps you would be interested in submitting a PR with these updates, @wittwerch?

**Comment by wittwerch** [comment]
Given we have capacity in the next weeks we'll put up a PR!

**Comment by benhaynes** [comment]
The style has been updated, but we should remove the ability to mention groups and simplify the syntax to: `@[1 Ben Haynes]`
